internal/graph/graph.go

```diff
@@ -213,3 +214,47 @@ func (g *Graph[V]) TopologicallySortWithPriority(isLowerPriority func(V, V) bool
 
 	return output, nil
 }
+
+func (g *Graph[V]) EncodeDOT(w io.Writer, sortVertices bool) (err error) {
+	builder := &dotBuilder{w}
+	err = builder.init()
```

Comment:
Combine these lines into 
```
if err := &builder.init(); err != nil {
   return err
}
```

Alternatively, you could do the init in a constructor for the builder